### DISCUSSION TITLE
How translate query with inner select into sea orm dsl?

### DISCUSSION THREAD

**Discussion by sergemrzn** [SeaQL/sea-orm#1969] [Q&A]
#help

Hello guys
How to translate this SQL
```SQL
SELECT event_id, created_at, data
FROM events
WHERE created_at > (SELECT created_at FROM events WHERE event_id = $1::uuid LIMIT 1)
ORDER BY created_at;
```
(event_id: uuid, created_at:date, daa: jsonb )
To single sea orm query?

**Comment by tyt2y3** [comment]
Not an exact answer, but you can take a look at https://www.sea-ql.org/SeaORM/docs/advanced-query/subquery/
